scripts/ci/check_compliance.py

```diff
@@ -844,6 +845,9 @@ def check_no_undef_outside_kconfig(self, kconf):
                               # Zephyr toolchain variant and therefore not
                               # visible to compliance.
         "BOARD_", # Used as regex in scripts/utils/board_v1_to_v2.py
+        "BOARD_MPS2_AN521_CPUTEST", # Used for board and SoC extension feature
+        "BOARD_NATIVE_SIM_NATIVE_64_TWO", # Used for board and SoC extension feature
+        "BOARD_NATIVE_SIM_NATIVE_ONE", # Used for board and SoC extension feature
```

Comment:
nit: Specifically, they're used for board and SoC extension feature _tests_.

Reply:
fixed